appConfig.js

```diff
@@ -2,6 +2,7 @@ const config = {
   port: 3001,
   webpackDevServerPort: 3000,
   baseUrl: '/books-music-movies/recommendations/best-books/',
+  base404: '/books-music-movies/recommendations/best-books/404',
```

Comment:
I think it is OK. But if we are adding it to the config, then we should just use it from config instead of typing it whenredirecting 404 page. I haven't check all the files yet so please ignore me if you have already done it in this PR :-P